### ISSUE TITLE
Does not detect the animation in this file

### ISSUE THREAD

**Issue by rosskarchner** [torchbox/Willow#45]
Hello! has_animation seems to be failing on this image, which is definitely animated:

![20160927_blog_1million](https://cloud.githubusercontent.com/assets/235397/18891499/7802fa40-84d3-11e6-889f-cee9dec55c04.gif)

Using 0.3.1:

```
>>> from willow.image import Image as WillowImage
>>> gif = open('/Users/karchnerr/Downloads/20160927_Blog_1million.gif')
>>> image = WillowImage(gif)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object() takes no parameters
>>> image = WillowImage.open(gif)
>>> image.has_animation
<function wrapper at 0x10154d938>
>>> image.has_animation()
False
```

I'm on a mac, but have confirmed this on Amazon Linux as well. Is there anything else I could provide to help troubleshoot this?

**Comment by rosskarchner** [comment]
I think I found a hint:

https://github.com/torchbox/Willow/blob/master/willow/plugins/pillow.py#L36

**Comment by chimeno** [comment]
Are you using Pillow?, because Pillow always returns False.

**Comment by rosskarchner** [comment]
Yeah - if we have both Pillow and Wand installed, is there a way to force it to use Wand? 

On September 30, 2016 3:46:45 AM EDT, Daniel Chimeno <notifications@github.com> wrote:
>Are you using Pillow?, because Pillow always returns False.
>
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/torchbox/Willow/issues/45#issuecomment-250681467

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by gasman** [comment]
@rosskarchner Wand will be used automatically if available. However, you also need to ensure that the Imagemagick dependency is installed before installing wand, as it will silently ignore the Wand installation otherwise. (On Debian / Ubuntu, the relevant package is `libmagickcore-dev`.)
